src/vm/jitinterface.cpp

```diff
@@ -7338,6 +7338,17 @@ bool getILIntrinsicImplementationForUnsafe(MethodDesc * ftn,
         methInfo->options = (CorInfoOptions)0;
         return true;
     }
+    else if (tk == MscorlibBinder::GetMethod(METHOD__UNSAFE__SKIPINIT)->GetMemberDef())
```

Comment:
Could you please add this to src\tools\crossgen2\Common\TypeSystem\IL\Stubs\UnsafeIntrinsics.cs as well?

Reply:
Missed it. I see that it is there already.